Python/Batch/article_samples/mpi/linux_mpi_task_demo.py

```diff
@@ -191,7 +191,7 @@
     multi_task_helpers.create_pool_and_wait_for_vms(
         batch_client, _POOL_ID, _NODE_OS_PUBLISHER, _NODE_OS_OFFER,
         _NODE_OS_SKU, _POOL_VM_SIZE, _POOL_NODE_COUNT, start_task_cmdline,
-        pool_starttask_files, run_elevated=True)
+        pool_starttask_files, elevation_level='admin')
```

Comment:
`elevation_level=batchmodels.ElevationLevel.admin` instead (assuming batchmodels is aliased)?